docs/source/blocks/editcomponent.md

```diff
@@ -127,6 +127,43 @@ import { Icon } from '@plone/volto/components';
 </SidebarPortal>;
 ```
 
+### Default values support for block schema forms
+
+In the block schema definition, you can use the two additional properties for
+each field: `default` and `missing`. The `default` value is used to populate
+the field with an initial value. The `missing` value should be used when the field
+should always have a value (as required).
+
```

Comment:
Off topic rant: offering suggestions for narrative documentation with arbitrary line breaks sucks. This is one good reason to use one sentence per line.

Anyway, here's my suggestion:

```
In the block schema definition, you can use the two additional properties for each field: `default` and `missing`.

The `default` value is used to set an initial value for the field.
The user may override this value or clear it.

The `missing` value must be a valid value which becomes the field's value to be submitted when the user clears the field.
With `missing`, the field is optional.
Without `missing`, the field is required.
```